docs/Formatting.md

```diff
@@ -1,5 +1,9 @@
 ## Formatting
 
+### Indentation
+
+The standard `editor.insertSpaces` & `editor.tabSize` [format settings](https://code.visualstudio.com/docs/editor/codebasics#_indentation) are used for [Spaces vs Tabs](https://www.youtube.com/watch?v=SsoOG6ZeyUI), and how many. To indent XML differently than e.g. Java, configure these settings language specifically ([example](https://github.com/vorburger/vorburger-dotfiles-bin-etc/blob/31c1953e550a21133ae10e017bb18fe829bc3711/dotfiles/code/settings.json#L27)).
```

Comment:
please embed the example in this file. This doc file can be viewed in VS Code directly, in offline mode.

Reply:
Done!